README.md

```diff
@@ -112,7 +112,7 @@ To make a custom colorscheme, check out the [template](./colorschemes/template.g
 `-p`, `--percpu` Show each CPU in the CPU widget.  
 `-a`, `--averagecpu` Show average CPU in the CPU widget.  
 `-s`, `--statusbar` Show a statusbar with the time.  
-`-b`, `--battery` Show battery level widget (`minimal` turns off). [preview](./assets/screenshots/battery.png)
+`-b`, `--battery` Show battery level widget (`minimal` turns off). [preview](./assets/screenshots/battery.png)  
 `-i`, `--interface=NAME` Select network interface [default: all].
```

Comment:
I think it would be a good idea to let users know in the readme that many interfaces can be defined with CSVs and that interfaces can be ignored with `!`

The same documentation should also appear in the [cli usage text](https://github.com/cjbassi/gotop/blob/master/main.go#L65-L87)

Reply:
Done

Reply:
NICE
![image](https://user-images.githubusercontent.com/5367247/64637436-a4923800-d3b8-11e9-8717-84cbbacfb744.png)